src/prognostic_equations/eddy_diffusion_closures.jl

```diff
@@ -612,6 +699,63 @@ function turbulent_prandtl_number(params, ᶜN²_eff, ᶜstrain_rate_norm)
     return min(max(prandtl_nvec, eps_FT), Pr_max)
 end
 
+
+function turbulent_prandtl_number(p)
```

Comment:
It's a little confusing to give this non-pointwise function the same name as its pointwise counterpart above. I would suggest renaming this to something like `ᶜturbulent_prandtl_number`. You can also move the function definition above inside this function (as a closure) in order to keep everything together and avoid cluttering the namespace.